src/Resources/ResourceManager/Implementation/ResourceGroups/ExportAzureResourceGroupCmdlet.cs

```diff
@@ -93,63 +96,90 @@ public class ExportAzureResourceGroupCmdlet : ResourceManagerCmdletBaseWithApiVe
         [Parameter(Mandatory = false, HelpMessage = "Do not ask for confirmation.")]
         public SwitchParameter Force { get; set; }
 
+        /// <summary>
+        /// Gets or sets the API version.
+        /// </summary>
+        [CmdletParameterBreakingChange("ApiVersion", ChangeDescription = "Parameter is being deprecated without being replaced")]
+        [Parameter(Mandatory = false, HelpMessage = "When set, indicates the version of the resource provider API to use. If not specified, the API version is automatically determined as the latest available.")]
+        [ValidateNotNullOrEmpty]
+        public override string ApiVersion { get; set; }
+
         /// <summary>
         /// Executes the cmdlet.
         /// </summary>
         protected override void OnProcessRecord()
         {
             base.OnProcessRecord();
+            String contents;
+
             if (ShouldProcess(ResourceGroupName, VerbsData.Export))
             {
 
                 var resourceGroupId = this.GetResourceGroupId();
 
-                var apiVersion = this.ApiVersion ?? DefaultApiVersion;
-
-                var parameters = new ExportTemplateParameters
+                if (this.ApiVersion is null)
```

Comment:
Use `IsParameterBound()` to determine if a parameter is assigned by user
```suggestion
                if (this.IsParameterBound(c => c.ApiVersion))
```

Reply:
+1

---
In reply to: [510558296](https://github.com/Azure/azure-powershell/pull/13275#discussion_r510558296) [](ancestors = 510558296)